TeamCode/src/main/java/org/firstinspires/ftc/teamcode/utilities/ColorDetect.java

```diff
@@ -0,0 +1,52 @@
+package org.firstinspires.ftc.teamcode.utilities;
+
+import static org.firstinspires.ftc.robotcore.external.BlocksOpModeCompanion.telemetry;
+
+import android.graphics.Color;
+
+import com.acmerobotics.dashboard.config.Config;
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+import com.qualcomm.robotcore.hardware.ColorSensor;
+import com.qualcomm.robotcore.hardware.HardwareMap;
+
+@Config
+public class ColorDetect {
+    ColorSensor colorSensor;
+
+    public ColorDetect(HardwareMap hMap) {
+        this.colorSensor = hMap.get(ColorSensor.class, "color");
+    }
+
+    public int[] readRGB() {
```

Comment:
no javadoc comment 😔

Reply:
what is javadoc

Reply:
> what is javadoc

the thing you do with like the @return @params and all that stuff @varghs might explain better